client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -56,47 +57,14 @@ export class CheckoutThankYouHeader extends PureComponent {
 		titanAppsUrlPrefix: PropTypes.string.isRequired,
 		translate: PropTypes.func.isRequired,
 		upgradeIntent: PropTypes.string,
+		isRedesignV2: PropTypes.bool,
 	};
 
 	isSearch() {
 		const { purchases } = this.props;
 		return purchases?.length > 0 && purchases[ 0 ].productType === 'search';
 	}
 
-	getHeading() {
```

Comment:
Moved this to [get-heading.ts](https://github.com/Automattic/wp-calypso/pull/78931/files#diff-28daa95864f94a5364e44ec5e03d5d2f5fbdec7ec13def80ea0cafebde2ab8dd)